dds_cli/project_status.py

```diff
@@ -113,3 +113,35 @@ def update_status(self, new_status, deadline=None, is_aborted=False, no_mail=Fal
         )
 
         dds_cli.utils.console.print(f"Project {response_json.get('message')}")
+
+
+class ProjectBusyStatusManager(base.DDSBaseClass):
+    """Project Busy Status manager class."""
+
+    def __init__(
+        self,
+        no_prompt: bool = False,
+        token_path: str = None,
+    ):
+        """Handle actions regarding project busy status in the cli."""
+        # Initiate DDSBaseClass to authenticate user
+        super().__init__(
+            no_prompt=no_prompt,
+            method_check=False,
+            token_path=token_path,
+        )
+
+    # Public methods ###################### Public methods #
+    def get_busy_projects(self, list):
+        """Check if there are busy projects"""
+
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.PROJ_BUSY_ANY,
+            method="get",
+            headers=self.token,
+            json={"list": list},
+            error_message="Failed to get projects with busy status",
+        )
+
+        dds_cli.utils.console.print(f"list: {list}")
+        dds_cli.utils.console.print(f"response: {response_json.get('projects')}")
```

Comment:
```suggestion
        num_busy: int = response_json.get("num")
        if num_busy is None:
            <something has gone wrong, the information is not returned and cannot be displayed, raise exception>
        if num_busy:  # this covers all except 0
            projects: typing.Dict = response_json.get("projects")
            if not projects:
                <something has gone wrong... raise exception>
            
            # otherwise ...
            <table with public id and date updated as columns>
        else:  
            <print out that there are no busy projects>
```

Reply:
Printing a table using utils.create_table is tricky due to how return_info from the endpoint is structured. Therefore I am just printing rows with busy projects now.